### ISSUE TITLE
Publish new version with JsonPointerException typo fix

### ISSUE THREAD

**Issue by arusahni** [stefankoegl/python-json-pointer#42]
Thanks for the library - it works well with jsonpatch :-)

I noticed that, when a `JsonPointerException` is raised due to an invalid location, the error message says "location must *starts* with /". The plural "starts" is a typo.  This appears to have been fixed in `master`. I was wondering if the corrected version could be pushed out as a bugfix?  Right now, I'm catching and correcting it before re-raising.

```
$ python3
Python 3.7.6 (default, Jan 11 2020, 15:56:36)
[Clang 11.0.0 (clang-1100.0.33.16)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import jsonpointer
>>> jsonpointer.JsonPointer("/field")
<jsonpointer.JsonPointer object at 0x10a80afd0>
>>> jsonpointer.JsonPointer("field")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/aru/.virtualenvs/venv/lib/python3.7/site-packages/jsonpointer.py", line 182, in __init__
    raise JsonPointerException('location must starts with /')
jsonpointer.JsonPointerException: location must starts with /
>>> jsonpointer.__version__
'2.0'
```

**Comment by stefankoegl** [comment]
I just released version 2.1. I think this should be fixed, then.

**Comment by arusahni** [comment]
That fixed it, thank you!
